docs/tutorials/configure/scuttlebot.md

```diff
@@ -16,138 +16,219 @@ languages: [ "python", "go" ]
-
-Refer to [Installing Raspberry Pi OS on the Raspberry Pi](/installation/prepare/rpi-setup/#install-raspberry-pi-os), if necessary.
-
+- A Raspberry Pi with [Raspberry Pi OS 64-bit Lite installed]((/installation/prepare/rpi-setup/#install-raspberry-pi-os))
 - [A SCUTTLE Robot](https://www.scuttlerobot.org/shop/)
-- A USB camera (webcam)
+- A USB webcam
 
 ## Start configuring your robot
 
 1. Go to [the Viam app](https://app.viam.com).
-2. If you already created your robot in the app, navigate to its **Config** tab and skip to [Configuring the board](#configuring-the-board).
-3. Create an **Organization**.
-   If you already have an Organization, then this step is optional.
-   If you need help with organizations and locations, see our [guide to using the Viam app](/manage/fleet/).
-4. Create a **Location**.
-   If you already have a Location, then this step is optional.
-5. Create a **robot** and navigate to its **Config** tab.
-   We will stay in **Builder** mode for this tutorial (as opposed to **Raw JSON**).
-   ![A screenshot of the Viam app UI showing the CONFIG tab of a robot.](/tutorials/scuttlebot/createcomponent.png)
-
-{{% alert title="Tip" color="tip" %}}
-When naming components, remember to use consistent letter casing to avoid problems with "missing" components.
-{{% /alert %}}
-
-## Configuring the board
+1. Create a **robot** and follow the setup instructions until the robot successfully connects to the Viam app.
+1. Navigate to the robot's **Config** tab.
+   ![The Viam app UI showing the config tab of the robot.](/tutorials/scuttlebot/createcomponent.png)
 
-Add your first component, the [board](/components/board/) (in this case the Raspberry Pi).
+## Configure the board
 
-1. Enter a name for your board in the **Name** field.
-   In this tutorial, we've named the board "local."
-   As long as you're consistent, you can name the board whatever you want.
-2. Select the component **Type**, "board."
-3. Select "pi" from the **Model** drop-down.
-4. Click **Create Component** and the board component panel will expand.
+Add your first component, the [board](/components/board/):
 
-We don't need to worry about any other attributes for this component.
-![Screenshot of the component configuration panel for a board component. The name (local), type (board) and model (pi) are shown at the top of the panel. No other attributes are configured.](/tutorials/scuttlebot/board-empty-json.png)
+{{< tabs name="Configure an pi Board" >}}
+{{% tab name="Config Builder" %}}
 
-## Configuring the motors
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter `local` as the name for your board, select the type `board`, and select the `pi` model.
 
-### Adding the right motor
+Click **Create component**.
 
-The next step is to add a motor and make it spin a wheel.
+![The component configuration panel for a board component.](/tutorials/scuttlebot/board-empty-json.png)
```

Comment:
[opt] I think it would be a lot better if this image showed the fully configured board instead of just partially configured. Folks might miss the one line right under it and it'd be nice to see what it should all look like.